app/Http/Controllers/PaymentController.php

```diff
@@ -411,7 +411,7 @@ public function update(UpdatePaymentRequest $request, Payment $payment)
     {
         if($request->entityIsDeleted($payment))
             return $request->disallowUpdate();
-        
+
         $payment = $this->payment_repo->save($request, $payment);
```

Comment:
This needs to be $request->all() also.

Reply:
Should be fine now 👍

![image](https://user-images.githubusercontent.com/13711415/72611905-8ec6ed80-392b-11ea-8fbc-9a2076a8d844.png)